File: 8kyu/counting-sheep/counting-sheep.py
# def count_sheeps(sheep):
#   return sum(filter(lambda x: x, sheep))

def count_sheeps(sheep):
  return sheep.count(True)



arr = [True,  True,  True,  False,
  True,  True,  True,  True ,
  True,  False, True,  False,
  True,  False, False, True ,
  True,  True,  True,  True ,
  False, False, True,  True]

print(count_sheeps(arr))

# print(True == 1)
# print(False == 0)

File: 8kyu/counting-sheep/counting-sheep.ts
export function countSheeps(arrayOfSheep: (boolean | undefined | null)[]) {
	return arrayOfSheep.filter(Boolean).length;
}

const arr = [
	true,
	true,
	true,
	false,
	true,
	true,
	true,
	true,
	true,
	false,
	true,
	false,
	true,
	false,
	false,
	true,
	true,
	true,
	true,
	true,
	false,
	false,
	true,
	true,
];

console.log(countSheeps(arr));
